apiClientDotNet/Clients/MessageClient.cs

```diff
@@ -71,7 +71,7 @@ public List<InboundMessage> getMessagesFromStream(String streamId, int since, in
             HttpWebResponse resp = restRequestHandler.executeRequest(null, url, false, WebRequestMethods.Http.Get, symConfig, true);
```

Comment:
Can you wrap restRequestHandler.executeRequest in a using statement? and then we dont need "resp.Close();"